var/spack/repos/builtin/packages/python/package.py

```diff
@@ -120,6 +121,7 @@ class Python(AutotoolsPackage):
     depends_on('libffi', when='+ctypes')
     depends_on('tk', when='+tkinter')
     depends_on('tcl', when='+tkinter')
+    depends_on('tix', when='+tix')
```

Comment:
Are there no configure flags to enable this? Does `+tix` require `+tk`?

Reply:
There are no flags, it is enabled by default. However, the Tix module will fail to run if it can't find the tix library at run time. Tix does require tcl and tk.

Reply:
Maybe add a:
```python
conflicts('+tix', when='~tk', msg='Tix requires tk')
```

Reply:
do you mean to conflict with ~tkinter?